packages/server/src/manifest-builder.ts

```diff
@@ -21,14 +21,31 @@ interface ManifestBuilderOptions {
   distDir: string;
 };
 
+function* updateSourceMapURL(filePath: string, sourcemapName: string){
+  let { size } = fs.statSync(filePath);
+  let readStream = fs.createReadStream(filePath, {start: size - 16});
+  readStream.on('data', currentURL => {
+    if(currentURL == '/manifest.js.map'){
+      fs.truncate(filePath, size - 16, () => {
+        fs.appendFileSync(filePath, sourcemapName);
+      })
+    }
```

Comment:
We've handled the happy path of finding what we're looking for, what's to be done if we _don't_ find the sourcemapping URL?

Reply:
Since `soucemapName` is `${fileName}.map`, would it be appropriate to do
```
if(currentURL == '/manifest.js.map' && sourcemapName != 'undefined.map' ){
  yield truncate(filePath, size - 16);
  fs.appendFileSync(filePath, sourcemapName);
};
```

Reply:
I don't think so. I'm wondering what happens in an `else` branch of this `if` statement.

Reply:
Once I'm finished writing tests, what _could_ we do in the `else` branch?

Reply:
If we don't find the sourcemapping url, then what does that mean?

Reply:
Oh you mean the URL from within `manifest.js`? The two cases I can think of would be if the `manifest.js` is missing from `dist/` or the manifest is built differently than what we're expecting.

Reply:
Yeah, those are the two cases I can think of too. If the `manifest.js` is missing, then the copy from the `build` -> `dist` will fail earlier on, so we can be relatively safe in the assumption that it is there. My question is what should we do when the manifest is built differently than what we're expecting?

Reply:
Can we rely on the tests to catch that or do we need something in between for cases when manifests gets built correctly in tests but not in production?

Reply:
We have to handle this case in production, since it indicates something is very wrong.